src/packages/core/src/servers/http-server.ts

```diff
@@ -71,6 +71,7 @@ function prepareCORSResponseHeaders(method: HttpMethods, request: HttpRequest) {
 
 function sendResponse(
   response: HttpResponse,
+  closeConnection: boolean,
```

Comment:
Instead of increasing the number of parameters passed to this function, can we just use `this.#isClosing` within `sendResponse`?

Reply:
We can't; it is a global function.